core/src/test/scala/integration/kafka/api/TransactionsBounceTest.scala

```diff
@@ -130,13 +129,10 @@ class TransactionsBounceTest extends KafkaServerTestHarness {
         iteration += 1
       }
     } finally {
-      producer.close()
-      consumer.close()
```

Comment:
We are now relying on the lifecycle methods to close the producer and consumer. Did you do that to simplify the code, i.e. it has no impact on the behavior of the test?

Reply:
Yeah, I thought it is a preferred pattern to simplify cleanup a little bit in test cases. It does not change the behavior of the test.